lib/db/mysql/patches/patch-002-001.sql

```diff
@@ -0,0 +1,9 @@
+--  Drop all the tables.
+--  (commented out to avoid accidentally running this in production...)
+
+-- DROP TABLE avatar_providers;
+-- DROP TABLE avatars;
+-- DROP TABLE avatar_selected;
+
```

Comment:
I usually try and make sure all statements in the reverse patch are the exact same reverse order as in the forward patch. This only makes sense when latter statements depend on earlier statements.

For example, does MySql complain here when `avatars` is dropped, but is still referenced from `avatar_selected`? And if not, it should! :)